module/batch-action/src/Domain/Command/ReprocessBatchActionCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Domain\Command;
+
+use Ergonode\BatchAction\Domain\Entity\BatchActionId;
+use Ergonode\SharedKernel\Domain\DomainCommandInterface;
+
+class ReprocessBatchActionCommand extends AbstractPayloadCommand implements DomainCommandInterface
+{
+    private BatchActionId $id;
+
+    private bool $autoEndOnErrors;
+
+    /**
+     * @param mixed $payload
+     */
+    public function __construct(
+        BatchActionId $id,
+        $payload = null,
+        bool $autoEndOnErrors = true
```

Comment:
Something off here - value can be nullable if not passed - then should be taken from the current action - this is not done in controller so I assume there's an error somewhere.